### PR TITLE
DOCS-695: Make preview gif and remove unneccessary image alt on bedtime songs bot tutorial

### PR DESCRIPTION
**Pull request by sguequierre**
Fixup from Naomi! 

* add preview "images" line and add gif

### CODE REVIEW

File: docs/tutorials/projects/bedtime-songs-bot.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight: 60
 type: "docs"
 tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
 description: "Create a robot babysitter with a webcam and machine learning."
-imageAlt: "Image of Tess and their computer."
+images: [ "/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```
Comment (sguequierre) [inline docs/tutorials/projects/bedtime-songs-bot.md:8]:
```suggestion
images: [ "../../tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```

**Comment by sguequierre** [comment]
i still can't get the path right to the assets

File: docs/tutorials/projects/bedtime-songs-bot.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ weight: 60
 type: "docs"
 tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
 description: "Create a robot babysitter with a webcam and machine learning."
-imageAlt: "Image of Tess and their computer."
+images: [ "../../tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```
Comment (sguequierre) [inline docs/tutorials/projects/bedtime-songs-bot.md:8]:
```suggestion
images: [ "/assets/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```

File: docs/tutorials/projects/bedtime-songs-bot.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ weight: 60
 type: "docs"
 tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
 description: "Create a robot babysitter with a webcam and machine learning."
-imageAlt: "Image of Tess and their computer."
+images: [ "/assets/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```
Comment (sguequierre) [inline docs/tutorials/projects/bedtime-songs-bot.md:8]:
```suggestion
images: [ "assets/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```

File: docs/tutorials/projects/bedtime-songs-bot.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ weight: 60
 type: "docs"
 tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
 description: "Create a robot babysitter with a webcam and machine learning."
-imageAlt: "Image of Tess and their computer."
+images: [ "assets/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```
Comment (npentrel) [inline docs/tutorials/projects/bedtime-songs-bot.md:8]:
```suggestion
images: [ "/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```
and it unfortunately needs to go in the static folder. Only images go into the assets folder - hugo does some resizing on them and then puts them into a folder in the deployed page. But it only does that for images

File: docs/tutorials/projects/bedtime-songs-bot.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ weight: 60
 type: "docs"
 tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
 description: "Create a robot babysitter with a webcam and machine learning."
-imageAlt: "Image of Tess and their computer."
+images: [ "/tutorials/img/bedtime-songs-bot/robot_babysitter.gif" ]
```
Comment (npentrel) [inline docs/tutorials/projects/bedtime-songs-bot.md:8]: you need to move the gif into the folder `static/tutorials/img/bedtime-songs-bot` then if you change it to this

**Review by npentrel — APPROVED** [review]

**Pull request merged**